StrategyGame/Model/Heroes/Peasant.cs

```diff
@@ -1,9 +1,37 @@
-﻿namespace StrategyGame.Model.Heroes
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using StrategyGame.Annotations;
+
+namespace StrategyGame.Model.Heroes
 {
     public class Peasant : IHero
     {
-        public int Hitpoints { get; set; }
-        public int Damage { get; set; }
-        public int Speed { get; set; }
+        public int Hitpoints {
+            get { return Hitpoints; }
```

Comment:
please, do not write all that stuff in one line
get
{
     return someValue;
}